modules/ROOT/pages/fd-connection-edit.adoc

```diff
@@ -0,0 +1,31 @@
+= Edit a Connection
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can edit a connection that is being used by or is available to be used by a specific type of connector, changing any of the information that you have specified in it.
+
+To edit a connection, you must log into Anypoint Platform with the user ID that was used to create the connection. You cannot edit a connection that was created under a user ID that you are not logged in with.
+
+If the connection is shared because you selected the *Share this connection with my business group* checkbox when you created it, the changes affect all of the connectors of that type in all of the projects in your business group.
+
+*To edit a connection:*
```

Comment:
DITTO prev. question about standards for the actual task steps: subtitle, bullet-list item, inline bold heading, or part of the main/only narrative body?